libs/ngx-sherfire/src/lib/firestore/sherfire-firestore.service.ts

```diff
@@ -0,0 +1,22 @@
+import { inject, Injectable, NgZone } from '@angular/core';
+import type { Firestore } from 'firebase/firestore';
+import { connectFirestoreEmulator, initializeFirestore } from 'firebase/firestore';
+import { SherfireApp } from '../app';
+import { FirebaseConfig } from '../firebase-config.service';
+import { NgxSherfireModule } from '../ngx-sherfire.module';
+
+export interface SherfireFirestore extends Firestore {}
+
+@Injectable({
+    providedIn: NgxSherfireModule,
+    useFactory: () =>
```

Comment:
This all used to happen in `firebaseFirestoreFactory` from `firebase-firestorm.service.ts`

Reply:
> `firebase-firestorm`

🔥🌪️